tests/ApiApprovalTests/net5+netcoreapp31/GraphQL.Server.Transports.AspNetCore.approved.txt

```diff
@@ -109,7 +110,6 @@ namespace GraphQL.Server.Transports.AspNetCore
         public bool ReadExtensionsFromQueryString { get; set; }
         public bool ReadQueryStringOnPost { get; set; }
         public bool ReadVariablesFromQueryString { get; set; }
-        public string ResponseContentType { get; set; }
```

Comment:
Not a breaking change; this property has not been released yet.

Reply:
ok